src/ImageProcessor/Imaging/ResizeLayer.cs

```diff
@@ -62,107 +46,152 @@ public class ResizeLayer
             this.Upscale = upscale;
             this.ResizeMode = resizeMode;
             this.AnchorPosition = anchorPosition;
-            this.CenterCoordinates = centerCoordinates ?? new float[] { };
+            if (centerCoordinates != null && centerCoordinates.Length == 2)
+            {
+                this.Center = new PointF(centerCoordinates[1], centerCoordinates[0]);
+            }
             this.MaxSize = maxSize;
-            this.RestrictedSizes = restrictedSizes ?? new List<Size>();
+            this.RestrictedSizes = restrictedSizes;
```

Comment:
No need to initialize this to an empty list, right? Implementations that expect a non-null value would already be susceptible to `NullReferenceException`s, as the property could be set to `null` after initialization.

Reply:
It could be set to null by a third part but that would be silly. You could add a backing field to the property to easily fix that though.

Reply:
The property is only used in `Resizer` and that already has a null-check: https://github.com/JimBobSquarePants/ImageProcessor/blob/4af8de4a6376d9426fadd5c4596bc1de187899c0/src/ImageProcessor/Imaging/Resizer.cs#L536

This is the kind of breaking change that won't cause major problems (although it should be documented in the release notes), but improves performance, as it doesn't need to construct the empty list 😉